lib/rails/instrumentation/subscribers/action_controller_subscriber.rb

```diff
@@ -103,13 +103,22 @@ def process_action(event)
             'span.kind' => 'server'
           )
 
+
+          status_code = event.payload[:status]
+          if status_code.is_a? Integer and status_code >= 500
+            tags['error'] = true
```

Comment:
you're setting `error` is the status code is 5xx and `exception` if an exception was thrown in an action, is that intended?

Reply:
`Utils.tag_error()` also sets `error=True` internally so we are setting error=true when either the status code is 5xx or an exception is raised (or both). In case an exception is raised, we also record it as span logs.